From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 13:39:17 -0700
Subject: fix-lib-install-dir.diff

01_qscintilla.pro.dpatch by Torsten Marek <shlomme@debian.org>
Don't install the library directly
Patch-Name: fix-lib-install-dir.diff
---
 Qt4Qt5/qscintilla.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Qt4Qt5/qscintilla.pro b/Qt4Qt5/qscintilla.pro
index 35b37da..6bc79d7 100644
--- a/Qt4Qt5/qscintilla.pro
+++ b/Qt4Qt5/qscintilla.pro
@@ -72,7 +72,7 @@ unix:!macx {
 # Scintilla namespace rather than pollute the global namespace.
 #DEFINES += SCI_NAMESPACE
 
-target.path = $$[QT_INSTALL_LIBS]
+target.path = /usr/lib
 INSTALLS += target
 
 header.path = $$[QT_INSTALL_HEADERS]
